R/lockfile.R

```diff
@@ -174,17 +174,27 @@ renv_lockfile_create <- function(project,
 
   lockfile <- renv_lockfile_init(project)
 
-  renv_lockfile_records(lockfile) <-
-
-    renv_snapshot_libpaths(libpaths = libpaths,
-                           project  = project) %>%
-
-    renv_snapshot_filter(project  = project,
-                         type     = type,
-                         packages = packages,
-                         exclude  = exclude) %>%
+  repeat {
```

Comment:
Can we add a comment here indicating the intention here is to allow the restart below to recompute the lockfile?